SportsHub/SportsHub.Api/Program.cs

```diff
@@ -66,6 +73,9 @@
     });
 });
 
+builder.Services.AddDbContext<ApplicationDbContext>(options => options.UseSqlServer(configuration.GetConnectionString("DefaultConnectionString")));
```

Comment:
I think we need to add some constant file, because we have too many custom strings within the project

Reply:
Can this be done at a later stage?

Reply:
yes, but we need to not forget about it